### PR TITLE
Fixes #29292 - Move pagination options to the client

### PR DESCRIPTION
**Pull request by tbrisker**

### CODE REVIEW

**Comment by tbrisker** [comment]
Please squash on merge and see GH-7497 for previous comments.

**Comment by tbrisker** [comment]
test failures are all intermittent and unrelated.

**Review by amirfefer — CHANGES_REQUESTED** [review]
Thanks @tbrisker , tested and works well! 
Though katello needs an update as well - https://github.com/Katello/katello/blob/a046c3b1ce8ff2ef5cc1da6f35b8f0cd02bcbfce/webpack/components/Table/TableWrapper.js#L6

**Comment by tbrisker** [comment]
Good find @amirfefer ! Opened https://github.com/Katello/katello/pull/9052 to fix it in katello. Next step might be to move it into the Pagination component itself so consumers don't need to care about it, but let's first get this change in.

File: webpack/assets/javascripts/react_app/routes/Audits/AuditsPage/components/AuditsTable.js [hunk 1]
```diff
@@ -4,35 +4,30 @@ import { withRenderHandler } from '../../../../common/HOC';
 import AuditsList from '../../../../components/AuditsList';
 import AuditsLoadingPage from './AuditsLoadingPage';
 import Pagination from '../../../../components/Pagination/Pagination';
-import { usePaginationOptions } from '../../../../Root/Context/ForemanContext';
```
Comment (tbrisker) [inline webpack/assets/javascripts/react_app/routes/Audits/AuditsPage/components/AuditsTable.js:7]: Just realized we don't actually need to pass this into the component, as it already defaults to it.

**Comment by tbrisker** [comment]
@amirfefer can you take another look? We'll need to merge katello one right after this one. Looks like the reason it is needed in katello is that they use patternfly 4 pagination which expects the options in a bit different manner.

**Review by amirfefer — APPROVED** [review]
Thanks @tbrisker :+1: 
 works well

**Pull request merged**